venmo-sdk/Venmo.m

```diff
@@ -298,6 +298,22 @@ - (void)validateAPIRequestWithCompletionHandler:(VENGenericRequestCompletionHand
 }
 
 
+#pragma mark - Friends
+
+- (void) getFriendsWithLimit:(NSNumber *)limit beforeUserID:(NSString *)beforeUserID afterUserID:(NSString *)afterUserID completionHandler:(VENGenericRequestCompletionHandler)handler {
+    
+    [self validateAPIRequestWithCompletionHandler:handler];
+    
+    NSMutableDictionary* parameters = [NSMutableDictionary dictionaryWithDictionary:@{@"access_token": self.session.accessToken, @"before": beforeUserID, @"after": afterUserID, @"limit": limit}];
```

Comment:
Additionally, `VENCore` will handle the access token if it is set with `VENCore -setAccessToken:`. See here: https://github.com/venmo/VENCore/blob/211d8fdd9e5489f7d27caa3bf12fb37521223ded/VENCore/Networking/VENHTTP.m#L221-L244